prow/pluginhelp/pluginhelp.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package pluginhelp defines structures that represent plugin help information.
+// These structs are used by sub-packages 'hook' and 'externalplugins'.
+package pluginhelp
+
+// PluginHelp is a serializable representation of the help information for a single plugin.
+// This includes repo specific configuration for every repo that the plugin is enabled for.
+type PluginHelp struct {
+	// Description is a description of what the plugin does and what purpose it achieves.
+	Description string
+	// WhoCanUse is a description of the permissions/role/authorization required to use the plugin.
```

Comment:
These are Github permissions, right? Please update the comment to reflect it.

Reply:
They are usually github permissions, but they can also be a github team (like the milestone status plugin) or potentially an OWNERS file alias. I updated the comment to clarify this.

Reply:
Well, github teams are an still github permissions but anyway :)